spinnaker/minimise.c

```diff
@@ -0,0 +1,154 @@
+/* SpiNNaker routing table minimisation.
+ *
+ * Minimises a routing table loaded into SDRAM and loads the minimised table
+ * into SDRAM with the specified application ID.
```

Comment:
Probably also worth noting that an RTE can occur for two reasons:
  1. RTE_MALLOC For running out of memory.
  2. RTE_ABORT If the table couldn't be sufficiently minimised.